app/domain/authentication/authn_jwt/configuration_factory.rb

```diff
@@ -1,13 +1,17 @@
 module Authentication
   module AuthnJwt
-    # Factory that receives a vendor name and returns the appropriate JWT vendor configuration class
+    # Factory that receives an authenticator name and returns the appropriate JWT vendor configuration class
     class ConfigurationFactory
```

Comment:
Maybe we make this class command class to like the other two factories?

Reply:
Yes but in the last ticket of the integration all parts